### DISCUSSION TITLE
adventures/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#39] [General]
# adventures/

https://shane0.github.io/adventures/

**Comment by shane0** [comment]
# dream vehicles

- https://www.tatratrucks.com/trucks/product-catalog/tatra-phoenix/

## roof tent

- https://antishanty.com/rooftop-camper/
- $10000 I think I can build a better one from scratch for that

The length of the roof of a Jeep TJ is 66 inches. This is measured from the front of the windshield to the back of the roof. The width of the roof is 59 inches.

The AntiShanty Rooftop Dwelling is available in two models: the RTD Slimline and the RTD Standard. The RTD Slimline is 72 inches long, 55 inches wide, and 11 inches high. The RTD Standard is 82 inches long, 55 inches wide, and 11 inches high.

Here is a more detailed breakdown of the dimensions of each model:

* **RTD Slimline:**
    * Length: 72 inches
    * Width: 55 inches
    * Height: 11 inches
    * Weight: 1,500 pounds
    * Cargo capacity: 28 cubic feet
* **RTD Standard:**
    * Length: 82 inches
    * Width: 55 inches
    * Height: 11 inches
    * Weight: 2,000 pounds
    * Cargo capacity: 36 cubic feet

Both models of the AntiShanty Rooftop Dwelling can sleep up to two people. They are both made from high-quality materials and are built to last. They are both also easy to install and use.

If you are looking for a hard-sided rooftop tent that doubles as a cargo box, the AntiShanty Rooftop Dwelling is a great option. It is well-made, easy to use, and has a large cargo capacity.

**Comment by shane0** [comment]
# glamping

- took out the springbar tent at middle forks
- [glamping pics](https://www.instagram.com/p/Crr9kJpuLRV/?utm_source=ig_web_copy_link&igshid=MzRlODBiNWFlZA==)

**Comment by shane0** [comment]
# washington

## section j

| Highlight | Miles |
|---|---|
| Stevens Pass | 0 |
| Kendall Katwalk | 6 |
| Spectacle Lake | 11 |
| Source Lake | 12 |
| Gem Lake | 13 |
| Upper Wildcat Lake | 14 |
| Wolverine Pass | 17 |
| Surprise Lake | 20 |
| Glacier Lake | 22 |
| Mosquito Pass | 34 |
| Lake Dorothy | 37 |
| Lemah Creek | 39 |
| Lemah Meadows | 42 |
| Waptus River | 45 |
| Cle Elum River | 48 |
| Deep Lake | 51 |
| Trap Lake | 54 |
| Joe Lake | 57 |
| Deception Pass | 60 |
| Park Lakes | 63 |
| Escondido Ridge | 66 |
| Stevens Pass East | 70 |

| Start | End | Altitude Change | Pass/Lake | Elevation |
|---|---|---|---|---|
| Stevens Pass | Kendall Katwalk | 2,000 ft | Kendall Katwalk | 4,860 ft |
| Kendall Katwalk | Spectacle Lake | 500 ft | Spectacle Lake | 4,360 ft |
| Spectacle Lake | Source Lake | 100 ft | Source Lake | 4,260 ft |
| Source Lake | Gem Lake | 200 ft | Gem Lake | 4,060 ft |
| Gem Lake | Upper Wildcat Lake | 300 ft | Upper Wildcat Lake | 3,760 ft |
| Upper Wildcat Lake | Wolverine Pass | 1,500 ft | Wolverine Pass | 2,260 ft |
| Wolverine Pass | Surprise Lake | 200 ft | Surprise Lake | 2,060 ft |
| Surprise Lake | Glacier Lake | 500 ft | Glacier Lake | 1,560 ft |
| Glacier Lake | Mosquito Pass | 1,200 ft | Mosquito Pass | 380 ft |
| Mosquito Pass | Lake Dorothy | 300 ft | Lake Dorothy | 780 ft |
| Lake Dorothy | Lemah Meadows | 500 ft | Lemah Meadows | 1,280 ft |
| Lemah Meadows | Waptus River | 1,000 ft | Waptus River | 2,280 ft |
| Waptus River | Cle Elum River | 800 ft | Cle Elum River | 1,480 ft |
| Cle Elum River | Deep Lake | 300 ft | Deep Lake | 1,180 ft |
| Deep Lake | Trap Lake | 400 ft | Trap Lake | 880 ft |
| Trap Lake | Joe Lake | 200 ft | Joe Lake | 680 ft |
| Joe Lake | Deception Pass | 1,000 ft | Deception Pass | -320 ft |
| Deception Pass | Park Lakes | 200 ft | Park Lakes | -120 ft |
| Park Lakes | Escondido Ridge | 1,300 ft | Escondido Ridge | 1,180 ft |
| Escondido Ridge | Stevens Pass East | 1,000 ft | Stevens Pass East | 2,180 ft |
| Stevens Pass East | Stevens Pass | 0 ft | Stevens Pass | 4,860 ft |

**Comment by shane0** [comment]
# oregon

## coast

Section | Start | End | Highlights | Miles
-- | -- | -- | -- | --
Section 1 | Fort Stevens State Park | Oswald West State Park | Nehalem Bay State Recreation Area, Cape Lookout State Park, Haystack Rock | 13
Section 2 | Oswald West State Park | Tillamook River Bridge | Neahkahnie Mountain, Hug Point, Three Arch Rocks | 12
Section 3 | Tillamook River Bridge | Lincoln City | Devil's Punchbowl, Cape Perpetua Scenic Area, Spouting Horn | 18
Section 4 | Lincoln City | Waldport | Cape Foulweather, Heceta Head Lighthouse | 13
Section 5 | Waldport | Florence | Oregon Dunes National Recreation Area, Siletz Bay State Recreation Site | 22
Section 6 | Florence | Reedsport | Honeyman State Park, Coos Bay | 17
Section 7 | Reedsport | Bandon | Cape Arago State Park, Face Rock State Natural Area | 15
Section 8 | Bandon | Port Orford | Cape Blanco State Park, Natural Bridges State Park | 16
Section 9 | Port Orford | Cape Sebastian | Cape Sebastian State Scenic Corridor | 13
Section 10 | Cape Sebastian | California border | Brookings, Harris Beach State Park | 20

| Day | Start | End | Miles | Elevation Gain | Highlights |
|---|---|---|---|---|---|
| 1 | Astoria | Newport | 40 | 1,000 | Cross the Astoria-Megler Bridge. |
| 2 | Newport | Waldport | 40 | 500 | Visit the Oregon Coast Aquarium. |
| 3 | Waldport | Yachats | 40 | 200 | Hike to Cape Perpetua. |
| 4 | Yachats | Florence | 40 | 500 | Visit the Oregon Dunes National Recreation Area. |
| 5 | Florence | Coos Bay | 40 | 1,000 | Visit the Coos Bay Maritime Museum. |
| 6 | Coos Bay | Reedsport | 40 | 500 | Visit the Umpqua Discovery Center. |
| 7 | Reedsport | Brookings | 40 | 1,000 | Visit the Chetco River State Scenic Corridor. |

**Comment by shane0** [comment]
# training program chatgpt

When preparing for a thru-hike, incorporating a strength training program into your routine can be beneficial for building the necessary strength, stability, and endurance. Here's a sample strength training program that can complement your thru-hiking preparation:

1. Squats: Strengthening your lower body, particularly the quadriceps and glutes, is crucial for uphill climbs and descending steep terrain. Perform squats using bodyweight or add resistance with barbells or dumbbells.

2. Lunges: Similar to squats, lunges target the lower body muscles and help with balance and stability. Perform walking lunges or stationary lunges to work the quads, glutes, and hamstrings.

3. Step-ups: Mimicking the motion of climbing uphill, step-ups are excellent for strengthening your legs and improving balance. Use a bench, step, or stairs to perform step-ups, alternating legs.

4. Deadlifts: Deadlifts engage multiple muscle groups, including the lower back, glutes, hamstrings, and core. They simulate the action of lifting and carrying a heavy backpack. Start with light weights and focus on proper form.

5. Push-ups: Upper body strength is essential for carrying a pack and maintaining balance. Regular push-ups target the chest, shoulders, and triceps. You can modify the difficulty by performing incline or decline push-ups.

6. Pull-ups or Rows: Strengthening the back and biceps is crucial for maintaining good posture and upper body endurance. If you have access to a pull-up bar, aim for a few repetitions. Otherwise, use resistance bands or perform rows using dumbbells or a suspension trainer.

7. Core exercises: A strong core is essential for stability and balance. Include exercises such as planks, Russian twists, bicycle crunches, and mountain climbers to target the abdominal and lower back muscles.

8. Stability and balance exercises: Incorporate exercises that challenge your balance, such as single-leg squats, single-leg deadlifts, and standing on one leg. These exercises help develop stability and mimic the uneven terrain encountered during hiking.

Remember to start with lighter weights or bodyweight exercises and gradually increase the intensity and resistance over time. Allow for rest and recovery days between strength training sessions to prevent overtraining and injury. Consulting with a fitness professional or personal trainer can help tailor a program to your specific needs and goals. Additionally, always prioritize safety and proper form during your workouts.

**Comment by shane0** [comment]
https://youtu.be/gbmQRmuv88c
bars

**Comment by shane0** [comment]
https://fire.airnow.gov/

**Comment by shane0** [comment]
https://www.pcta.org/discover-the-trail/closures/washington/

**Comment by shane0** [comment]
# 2023 pct sections i & j

- todo: finish uploading pics and videos
- https://youtu.be/wTYTqTvHULg

**Comment by shane0** [comment]
# bear hang

- this year i switched from ursack to the mld food bag 
- [instructions](https://youtu.be/lUt2erhk3EM)

**Comment by shane0** [comment]
- jdm starlink build <https://youtu.be/lBdqdJPSJP0>

**Comment by shane0** [comment]
Fold wall tent
https://youtu.be/ite5CHeocc8?si=Dfhhk1yxN7erXTj-

https://youtu.be/ZyU0Qg41gYQ?si=wlI0GLjBmrpGY8NL

**Comment by shane0** [comment]
https://dutchcyclinglifestyle.com/

**Comment by shane0** [comment]
pacific coast
https://www.adventurecycling.org/routes-and-maps/adventure-cycling-route-network/pacific-coast/
